types/web3/utils.d.ts

```diff
@@ -35,13 +35,13 @@ type Mixed =
     | number
     | BigNumber
     | {
-    type: string;
-    value: string;
-}
+        type: string;
+        value: string;
+    }
     | {
-    t: string;
-    v: string;
-};
+        t: string;
+        v: string;
+    };
```

Comment:
Please do not add whitespace changes. @phra What is going on here? I have seen this been added recently and I do not understand why the linting goes through with both versions. Do you have an idea?

Reply:
@Levino My editor added the whitespace because .editorconfig (at the root of DefinitelyTyped) has:

`indent_size = 4`

Should this not be enforced?

Reply:
Exactly. Linting should fail on incorrect indentation. We should fix this with another PR.

Reply:
+1 on whitespace change revert. Please revert these lines.

Reply:
@zlumer My editor makes this change because it is conforming to the .editorconfig in the project, setting spacing to 4 spaces.  Lint is not complaining.  Why revert it?

Reply:
I suggest we fix the indentation in a separate PR, since it's unrelated to the issue at hand (`fromWei` fix).
Changing line indentation breaks `git blame`, making it harder to keep track of what's going on in the file, as well as polluting this PR's diff.
However, if @phra and @Levino don't have an issue with that, I'm open to approving this PR. The change is well-made and the only disagreement we have is code-style oriented.